api/v1alpha08/kogitoserverlessplatform_types.go

```diff
@@ -140,11 +140,11 @@ type KogitoServerlessPlatformStatus struct {
 	Info map[string]string `json:"info,omitempty"`
 }
 
+// KogitoServerlessPlatform is the Schema for the kogitoserverlessplatforms API
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:object:root=true
 // +kubebuilder:object:generate=true
 // +kubebuilder:subresource:status
-// KogitoServerlessPlatform is the Schema for the kogitoserverlessplatforms API
```

Comment:
Changing this order won't affect the CRD description and include the annotations there?

Reply:
Moved the description to the first field since the IDE was complaining that the struct have no description.